src/api/SuggestionApi.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2018 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import Qs from 'qs';
+import { SearchApi } from './SearchApi';
+
+/** Default suggestions request serializer */
+class InvenioSuggestionRequestSerializer {
+
+  constructor(queryField) {
+    this.queryField=queryField;
+  }
+  /**
+   * Return a serialized version of the app state `query` for the API backend.
+   * @param {object} stateQuery the `query` state to serialize
+   */
+  serialize = stateQuery => {
+    const { suggestionString } = stateQuery;
+
+    const getParams = {};
+    if (suggestionString !== null) {
+      getParams['q'] = this.queryField + ':' + suggestionString;
+    }
+
+    return Qs.stringify(getParams, { arrayFormat: 'repeat', encode: false });
+  };
+}
+
+/** Default suggestions response serializer */
+class InvenioSuggestionResponseSerializer {
+
+  constructor(responseField) {
+    this.responseFieldPath = responseField.split(".");
+  }
+
+  _extractSuggestion(hit) {
+    var currentObject = hit
+
+    this.responseFieldPath.forEach((field) => {
+      currentObject = currentObject[field]
+    });
+
+    return currentObject;
+  }
```

Comment:
Not really, because the path can be nested. E.g.

``` json
[
    { 
        "a": {
            "b": {
                "title": "value to extract"
            }
        }
   },
  { 
      "a": {
          "b": {
              "title": "value to extract"
          }
      }
  }
]
```

Therefore I have to loop in the object until I react the inner most (``title`` in the example). It can also not be accessed normally because we cannot assume how many levels of nesting might there be.

Reply:
@ppanero I think this works for you [lodash/get](https://lodash.com/docs/4.17.11#get) no?

Reply:
Yeay! Thanks!